### PR TITLE
force MEMORY_LOCK default to false.

### PR DESCRIPTION
**Pull request by mindw**
Depends on pires/docker-elasticsearch#40 .

### CODE REVIEW

**Comment by pires** [comment]
Did this work when you ran it?

**Comment by mindw** [comment]
We've used 1.x & 2.x without this flag on k8s without any issues.
Other deployments for k8s do not use this mode - k8s requires swap to be disabled.

**Comment by pires** [comment]
Did you test 5.x w/ on Kubernetes this flag set to `false`? I will need to test it to make sure it works.

**Comment by mindw** [comment]
Will do. It a bit of a pain due amount if images to build ...

**Comment by pires** [comment]
I cannot merge this without testing so I'm facing the same pain ;)

**Comment by mindw** [comment]
Sorry, was away, You're quite right.

I'll have this tested and report back.

**Comment by pires** [comment]
OK @mindw

**Comment by Dexyon** [comment]
@mindw What is the status of this issue? If there is a docker which I can use for testing I can also run some tests. I have a set-up which causes reboot of my nodes every few days (probably) because if this issue!

**Comment by mindw** [comment]
Still in my to do list :( - I'll try to get to this till the end of the week.

**Comment by Dexyon** [comment]
@mindw If you need any help, just let me know ! I am using this library as well, and the least thing I could do is contribute a bit more and help my fellow users out! :)

**Comment by mindw** [comment]
@pires @Dexyon,
I've verified that with the above change the ES cluster upgrade & forms without complaints.
As a bonus the security context all nodes can be dropped.

File: README.md [hunk 1]
```diff
@@ -16,7 +16,10 @@ See [pires/kubernetes-elasticsearch-cluster](https://github.com/pires/kubernetes
 ## Environment variables
 
 This image can be configured by means of environment variables, that one can set on a `Deployment`.
+`MEMORY_LOCK` defaults to `false` as in Kubernetes cluster swap is always 
+turned off.
```
Comment (pires) [inline README.md]: Please, move this down to the list of environment variables.

**Comment by mindw** [comment]
@pires done.

**Comment by abh** [comment]
I built an image and am using it now; it seems to work fine `https://quay.io/repository/abh/docker-elasticsearch-kubernetes:5.5.1-nomemlock`

**Comment by pires** [comment]
I'll keep this open until a new version of Elasticsearch comes out.

**Comment by pires** [comment]
Release in version 5.5.2.

**Pull request merged**